SemiTruck.h

```diff
@@ -1,2 +1,19 @@
 #pragma once
+#include "Vehicle.h"
 
+struct SemiTruck : public Vehicle
+{
+    SemiTruck(const std::string& n);
+
+    //Special member Functions.  See instruction 9) and note in main()
+    //virtual ~SemiTruck() = default;
+    SemiTruck(const SemiTruck&) = default;
+    SemiTruck& operator=(const SemiTruck&) = default;
+
+    void pullOver();
+
+    void honkBigHorn(); 
+
+    //needed a virtual function to get rid og the warning.
+    void setSpeed(int s) override;
```

Comment:
This is not how you solve the warning.  Also the base class already has a `setSpeed` function, and your implementation is IDENTICAL to the base class implementation.
- [ ] remove